lib/build_runner.zig

```diff
@@ -1014,6 +1046,7 @@ fn usage(builder: *std.Build, already_ran_build: bool, out_stream: anytype) !voi
         \\  --global-cache-dir [path]    Override path to global Zig cache directory
         \\  --zig-lib-dir [arg]          Override path to Zig lib directory
         \\  --build-runner [file]        Override path to build runner
+        \\  --seed [string]              Seed the prng used in the build runner
```

Comment:
```suggestion
        \\  --seed [string]              For shuffling dependency traversal order (default: 0)
```

Reply:
This help message is ambiguous to me, is the default value supposed to be the seed used when the argument is not present, or when only `--seed` is passed with no value (and what would be the use case then ?) ?
Since the build runner is rarely called directly, but instead driven by `zig build`, should the zig cli mirror this if applicable ?

Reply:
This usage text is the text printed by `zig build -h`, so it should be appropriate for those users. I made a mistake in the above suggestion. A better suggestion would be:

```
  --seed [integer]              Shuffles dependency traversal order (default: random)
```

The Zig CLI will choose a random seed if the user does not provide one explicitly. The build runner should use 0 if none is provided. However, in practice when using `zig build` a seed will always be provided.

Reply:
Okay, makes more sense thanks.